storage/aptosdb/src/state_kv_db.rs

```diff
@@ -101,7 +101,9 @@ impl StateKvDb {
         COMMIT_POOL.scope(|s| {
             let mut batches = sharded_state_kv_batches.into_iter();
             for shard_id in 0..NUM_STATE_SHARDS {
-                let state_kv_batch = batches.next().unwrap();
+                let state_kv_batch = batches
+                    .next()
+                    .expect("sharded state kv batches shouldn't be empty");
```

Comment:
This error is not accurate, it happens when the number of items is less than NUM_STATE_SHARDS.